views/partial/footer.ejs

```diff
@@ -1,2 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>PDA MIT</title>
+    <!-- Pre-build CSS -->
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+	<link href="http://fonts.googleapis.com/css?family=Cookie" rel="stylesheet" type="text/css">
+</head>
+<body>
+
+    <!--The text and break tags below are placed just for testing purpose. They can be removed later-->
+    <h1>Scroll down to view the footer</h1><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+
+    <footer class="footer-distributed">
+
+        <div class="footer-left">
+    
+            <img src="public\img\pda-logo.png" alt="logo" class="logo">
```

Comment:
`<img src="images\pda-logo.png" alt="logo" class="logo">`

Also fix the the logo image .. It looks shrinken 

![image](https://user-images.githubusercontent.com/53329034/147415561-f7b7a6da-dc4a-4f06-aafc-f25f88146c63.png)